frontend/app/types.d.ts

```diff
@@ -2,7 +2,7 @@ import { type ParseKeys } from 'i18next';
 
 import type common from '../public/locales/en/common.json';
 import type gcweb from '../public/locales/en/gcweb.json';
-import { type Breadcrumbs, type BuildInfo, type I18nNamespaces, type PageIdentifier, type PageTitle } from './utils/route-utils';
+import { type Breadcrumbs, type BuildInfo, type I18nNamespaces, type PageIdentifier, type PageTitleI18nKey } from './utils/route-utils';
```

Comment:
```suggestion
import { type Breadcrumbs, type BuildInfo, type I18nNamespaces, type PageIdentifier, type PageTitleI18nKey } from '~/utils/route-utils';
```

Reply:
🤷 --- I wonder if we can set up an eslint rule to enforce this.

Reply:
Fixed